api/v6/report_server.thrift

```diff
@@ -382,10 +382,24 @@ service codeCheckerDBAccess {
   string getPackageVersion();
 
   // remove bug results from the database
+  // !!! DEPRECATED !!!
+  // Use removeRun to remove the whole run or removeRunReports to remove
+  // filtered run results.
   // PERMISSION: PRODUCT_STORE
   bool removeRunResults(1: list<i64> runIds)
+
+  // remove bug results from the database
+  // PERMISSION: PRODUCT_STORE
+  bool removeRunReports(1: list<i64>    runIds,
+                        2: ReportFilter reportFilter,
+                        3: CompareData  cmpData)
                         throws (1: shared.RequestFailed requestError),
 
+  // Remove run from the database.
+  // PERMISSION: PRODUCT_STORE
+  bool removeRun(1: i64 runId)
```

Comment:
Do we want to support of the removal of multiple runs through this API?

Reply:
I don't think so. This way we can call this function asynchronously and we can handle warnings much better for example if a run is locked.